LICENSE

```diff
@@ -248,6 +248,8 @@ License: http://www.apache.org/licenses/LICENSE-2.0
 This product includes code from Presto.
 
 * Retry wait and jitter logic in Tasks.java
+* S3FileIO logic derived from PrestoS3FileSystem.java in S3InputStream.java
```

Comment:
At this point of time, I don't see much similarity between the input and output stream of `PrestoS3FileSystem` and the ones here. Should we still have this line in license?

Reply:
This was what we used as a reference, so even though the vast majority has changed, I feel it's still best to credit.